playwright-e2e/pages/family-history.ts

```diff
@@ -0,0 +1,141 @@
+import { expect, Page } from '@playwright/test';
+import PageBase from 'pages/page-base';
+import { booleanToYesOrNo } from 'utils/test-utils';
+import { CancerSelector } from './cancer-selector';
+
+
+/**
+ * CMI's family history
+ */
+export class FamilyHistory {
+
+  private readonly page:Page;
+
+/*
+
+await page.getByText('Understanding the history of cancer in someone\'s biological family can help rese').click();
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.locator('mat-card-content').filter({ hasText: 'Biological / Birth Parent 1: Assigned Female at birth 1 questions answered' }).getByRole('button', { name: 'Edit' }).click();
+  await page.getByTestId('answer:FH_PARENT1_ALIAS').fill('Mom');
+  await page.locator('.picklist-answer-FH_PARENT1_LIVING').getByText('Yes').click();
+  await page.locator('.picklist-answer-FH_PARENT1_AGE_RANGE').getByRole('combobox').selectOption('90-94');
+  await page.locator('.picklist-answer-FH_PARENT1_HAD_CANCER').getByText('Yes').click();
+
+*/
+
+constructor(page: Page) {
+    this.page = page;  
+  }
+
+  async waitForReady(): Promise<void> {
+    await this.page.getByText('Understanding the history of cancer in someone\'s biological family can help rese').click();
```

Comment:
small thing but maybe rese can be extended to be research?

Reply:
Yes--this is generator code doing truncation.